src/components/filterbar/VehicleInput.js

```diff
@@ -1,47 +1,89 @@
 import React from "react";
 import SuggestionInput from "./SuggestionInput";
 import flow from "lodash/flow";
-import flatMap from "lodash/flatMap";
-import unionBy from "lodash/unionBy";
+import get from "lodash/get";
 import {observer, inject} from "mobx-react";
 import {app} from "mobx-app";
+import styled from "styled-components";
 
-const getSuggestionValue = (suggestion) => suggestion;
+const SuggestionContent = styled.div`
+  display: flex;
+  align-items: center;
+  background: ${({isHighlighted = false}) =>
```

Comment:
ok I see, maybe we should implement these also in StopInput

Reply:
Yep, good idea!